metatron-app/metatron-discovery
# #2605 add filtering, sorting functionality in explore data list

add sorting, filtering function in explore-data-list

### Description

sorting by name, popularity, modified time
filtering by source type

**Related Issue** : 

#2605 

### How Has This Been Tested?

1. LNB -> Exploration -> data
2. search something
3. check filtering and sorting works properly
#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
